src/world/footpath.h

```diff
@@ -34,8 +34,9 @@ typedef struct rct_footpath_entry {
 	uint32 bridge_image;		// 0x06
 	uint8 var_0A;
 	uint8 flags;				// 0x0B
+	uint8 scrolling_mode;		// 0x0C
 } rct_footpath_entry;
-assert_struct_size(rct_footpath_entry, 12);
+assert_struct_size(rct_footpath_entry, 13);
```

Comment:
Ah, I get what you're saying. Should maybe add manual padding for packed arrays then?
As a side note, I thought unaligned memory access wasn't really a problem on x86 anymore, except for certain SIMD instructions?